css/style.css

```diff
@@ -0,0 +1,3117 @@
+}
+
+.road__image-stroke {
+  width: 29px;
+  height: 44.97px;
+  position: absolute;
+  left: 0px;
+  top: 6px;
+  z-index: 4;
+}
+
+.road__image-strokeDotted {
+  width: 26.4px;
+  height: 42.37px;
+  position: absolute;
+  left: .7px;
+  top: 7px;
+  z-index: 5;
+}
+
+.road__image-ellipseRightTop {
+  position: absolute;
+  width: 7px;
+  height: 7px;
+  z-index: 6;
+  left: 22px;
+  top: 5px;
+}
+
+.road__image-ellipseRightTopIn {
+  position: absolute;
+  width: 3px;
+  height: 3px;
+  left: 24px;
+  top: 7px;
+  z-index: 7;
+}
+
+.road__image-ellipseLeftBottom {
+  position: absolute;
+  width: 7px;
+  height: 7px;
+  left: 7px;
+  top: 46px;
+  z-index: 8;
+}
+
+.road__image-ellipsrLeftBottomIn {
+  position: absolute;
+  width: 3px;
+  height: 3px;
+  z-index: 9;
+  left: 9px;
+  top: 48px;
+}
+
+.road__title {
+  display: flex;
+  flex-direction: row;
+  font-family: "MontserratSemiBold";
+  align-items: center;
+  padding-left: 40px;
+}
+
+.road__title-numb {
+  font-size: 35px;
+  margin: 0px;
+}
+
+.road__title-text {
+  align-self: flex-end;
+  padding-bottom: 3px;
+  font-size: 20px;
+  margin: 0px;
+  padding-left: 6.5px;
+}
+
+.road__desc {
+  font-family: "MontserratRegular";
+  margin: 0px;
+  font-size: 12px;
+  padding-left: 40px;
+  margin-top: -3px;
+}
+
+.description-characteristic__testDrive {
+  display: flex;
+  align-items: center;
+  padding-right: 40px;
+}
+
+.testDrive__image {
+  position: relative;
+  padding-right: 40px;
+}
+
+.testDrive__image-ellipse {
+  position: absolute;
+  width: 22px;
+  height: 22px;
+  top: -11px;
+  left: 10px;
+}
+
+.testDrive__image-tesla {
+  position: absolute;
+  width: 14.18px;
+  height: 8.23px;
+  left: 14px;
+  top: -1px;
+  animation-duration: 3s;
+  animation-name: rotate;
+  animation-iteration-count: infinite;
+  animation-direction: alternate;
+  transform-origin: 50% 0%;
+}
+
+@keyframes rotate {
+  from {
+    transform: rotate(0deg);
+  }
+
+  to {
+    transform: rotate(-230deg);
+  }
+}
+
+.testDrive__image-dott {
+  position: absolute;
+  width: 2.58px;
+  height: 2.58px;
+  left: 20px;
+  top: -3px;
+}
+
+.testDrive__title {
+  margin: 0px;
+  padding-left: 0px;
+  font-size: 13px;
+  font-family: "MontserratBold";
+}
+
+.description-characteristic__button {
+  margin: 0px;
+  color: #E51937;
+  display: flex;
+  align-self: center;
+  font-family: "MontserratBold";
+  font-size: 13px;
+  border: 2px solid #E51937;
+  box-shadow: none;
+  transition: box-shadow 0.6s linear;
+  padding: 10px 40px;
+}
+
+.description-characteristic__button:hover {
+  box-shadow: 0px 0px 0px 2px #E51937;
+}
+
+.description-characteristic__testDrive-phone {
+  display: none;
+}
+
+.description-characteristic__button-phone {
+  display: none;
+}
+
+.characteristic__arrowDown {
+  display: flex;
+  justify-content: center;
+  padding-top: 37px;
+  position: relative;
+  padding-left: 70px;
+  animation: arrow 1s linear infinite;
+}
+
+@keyframes arrow {
+  0% {
+      bottom:0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/tesla_project/pullRequest?prid=5558646)